client/components/game-forms/EventForm.js

```diff
@@ -6,50 +7,57 @@ let EventForm = props => {
   const {handleSubmit, pristine, submitting} = props
 
   return (
-    <form onSubmit={handleSubmit}>
-      <label htmlFor="eventName">Event Name </label>
-      <Field name="name" component="input" type="text" placeholder="My Event" />
-      <label htmlFor="date">Date of Event</label>
-      <Field
-        name="date"
-        component="input"
-        type="date"
-        placeholder={Date.now()}
-      />
-      <label htmlFor="location"> Event Location</label>
+    <div className="container">
+      <form className="row" onSubmit={handleSubmit}>
+        <div className="col s12 m6 input-field">
+          <Field name="name" component="input" type="text" />
+          <label htmlFor="name"> First Name </label>
+        </div>
+        <div className="col s12 m6 input-field">
+          <Field
+            name="date"
+            component="input"
+            type="date"
+            placeholder={Date.now()}
+          />
+          <label htmlFor="date">Date of Event</label>
+        </div>
 
-      <Field
-        name="location"
-        component="input"
-        type="text"
-        placeholder="Chicago"
-      />
-      <label htmlFor="description">Event Description</label>
-      <Field
-        name="description"
-        component="input"
-        type="text"
-        placeholder="An Event in Chicago"
-      />
-      <label>Event Status</label>
-      <Field
-        name="status"
-        component="select"
-        ref={() => $('select').formSelect()}
-      >
-        <option value="pending">Pending</option>
-        <option value="in_progress">In Progress</option>
-        <option value="done">Done</option>
-      </Field>
-      <button
-        className="btn waves waves-light"
-        type="submit"
-        disabled={pristine || submitting}
-      >
-        {' '}
-        {props.formAction} This Event{' '}
-      </button>
-    </form>
+        <div className="col s12 m6 input-field">
+          <Field name="location" component="input" type="text" />
+          <label htmlFor="location"> Event Location</label>
+        </div>
+        <div className="col s12 input-field">
+          <Field name="description" component="input" type="text" />
+          <label htmlFor="description">Event Description</label>
+        </div>
+
+        <div className="col s12 input-field">
+          <Field
+            name="status"
+            component="select"
+            ref={() => $('select').formSelect()}
+          >
+            <option value="pending">Pending</option>
+            <option value="in_progress">In Progress</option>
```

Comment:
I don't think we need an option for the status of an event when we are creating it. It will always be pending. Maybe only for the edit?

Reply:
Sounds Great I'll remove